OasisDEX/oasis-borrow
# added adroll pixel script to _app

# [Add adRoll pixel to the landing page](https://app.shortcut.com/oazo-apps/story/2964/add-adroll-pixel-to-the-landingpage)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- added adRoll script to the Head component
  
## How to test 🧪
  <Please explain how to test your changes>
- use dedicated adRoll dashboard to check if instances of oasis.app (dev / staging / prod) are connected
- by using PixelPal extension for Chrome 
    
## Definition of done ✔️
